backend/src/bundles/users/user.model.ts

```diff
@@ -40,6 +46,14 @@ class UserModel extends AbstractModel {
                     to: `${DatabaseTableName.USER_DETAILS}.${UserDetailsAttributes.USER_ID}`,
                 },
             },
+            subscriptions: {
+                relation: Model.HasOneRelation,
```

Comment:
HasMany? you said that user could potentially have multiple subscription records in DB

Reply:
If you remove this relation from the model, it won't magically disappear, it will still exist but just won't be declared. Someone reading the code later might have a hard time determining the presence of this relation. So, I would say keep it to ensure our models closely represent the actual state of the database, even if you don't use it now.

Reply:
Ok. You're right!